extensions/reviewed/MarchingSquares.json

```diff
@@ -1981,7 +1979,7 @@
           "objectGroups": []
         },
         {
-          "description": "Cap every value of the field to a range.",
+          "description": "Restrict every value of the field to a range.",
```

Comment:
I chose "cap" because the values are cut. "Restrict" could mean anything, for instance something like a scaling.
Is "cap" too slang?

Reply:
EIther word works, I will keep your version.